controllers/openstack_ansibleee_controller.go

```diff
@@ -428,12 +422,16 @@ func (r *OpenStackAnsibleEEReconciler) jobForOpenStackAnsibleEE(ctx context.Cont
 
 // labelsForOpenStackAnsibleEE returns the labels for selecting the resources
 // belonging to the given openstackansibleee CR name.
-func labelsForOpenStackAnsibleEE(name string, deployIdentifier string) map[string]string {
-	return map[string]string{
+func labelsForOpenStackAnsibleEE(name string, labels map[string]string) map[string]string {
+	ls := map[string]string{
 		"app":                   "openstackansibleee",
-		"deployIdentifier":      deployIdentifier,
 		"openstackansibleee_cr": name,
+		"aee":                   "true",
```

Comment:
What is the aee label for? The shortname is actually osaee, is it meant to correspond to that?

Reply:
I just wanted to have a shorter label than `app=openstackansibleee`,
but you are right, better use the shortname